.github/workflows/scrape.yml

```diff
@@ -9,6 +9,13 @@ jobs:
   scrape:
     runs-on: ubuntu-latest
     name: Scrape data into JSON file with Python script
+    env:
+      TIMETABLE_PREFIX: http://timetabling.anu.edu.au/sws
+    strategy:
+      fail-fast: false
+      matrix:
+        year: [2022]
+        session: ['1 S1', '2 S2', '3 X1', '4 X2', '5 X3', '6 X4']
```

Comment:
What happens if one of the combinations in future doesn't exist? Is that what `fail-fast: false` is there for, to make it silently fail and continue with other combos?

Reply:
> Spot on

Huh not sure why my comment (above) didn't show up